src/vaadin-date-picker-mixin.html

```diff
@@ -690,7 +690,7 @@
       if (this._nativeInput && this._nativeInput.selectionStart) {
         this._nativeInput.selectionStart = this._nativeInput.selectionEnd;
       }
-      this.validate();
+      !this.value && this.validate();
```

Comment:
Please add a comment describing why we don't have to run validation if value is set.
Has it been validated before, in `_selectedDateChanged`?

Reply:
Added.